keras_retinanet/preprocessing/csv_generator.py

```diff
@@ -122,7 +121,8 @@ def image_aspect_ratio(self, image_index):
 
     def load_image(self, image_index):
         path = self.image_names[image_index]
-        return cv2.imread(path)
+        # Return image with channels reversed (BGR instead of RGB).
+        return np.asarray(Image.open(path))[:, :, ::-1]
```

Comment:
I don't mind if this is done for all generators. I'll make a PR for removing the opencv dependency in other generators and in the data augmentation code.

Reply:
I intended to add a utility function `utils.image.read_image_bgr()`, but then I noticed that there is still a opencv dependency there even if reading images is done with PIL.

Still, we should probably put it there, otherwise we're going to have `np.asarray(Image.open(...))[:, :, ::-1]` all over the place.

Reply:
Moved it to utils now.